packages/components/src/components/badge/badge.css

```diff
@@ -0,0 +1,57 @@
+:host {
+  --color-badge: #e20074;
+  --inner-width-small: 86px;
+  --inner-height-small: 88px;
```

Comment:
can you use more `--scl-spacing-` type design tokens instead of all these magic pixel value numbers?

Reply:
please take a look again here